PendingReleaseNotes.md

```diff
@@ -16,6 +16,7 @@
   - The on-host log directory for OSDs was set incorrectly to `<dataDirHostPath>/<namespace>/log`;
     fix this to be `<dataDirHostPath>/log/<namespace>`, the same as other daemons.
   - Use the mon configuration database for directory-based OSDs, and do not generate a config
+  - Support PersistentVolume backed by LVM Logical Volume for "OSD on PVC".
```

Comment:
Are you also adding support for partitions and encrypted devices? These are mentioned in your doc change above.

Reply:
As I mentioned above, partitions and encrypted devices before my PR.